index.html

```diff
@@ -41,8 +41,167 @@ <h1 class="primary">
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="project-container">
+                    <div class="img-container"></div>
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                    
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="project-container">
+                    <div class="img-container"></div>
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                    
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="project-container">
+                    <div class="img-container"></div>
+                    <div class="project-details">
+                        <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                    
+                        <ul class="tech-stack">
+                            <li class="tag">Ruby on Rails</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">JavaScript</li>
+                            <li class="tag">HTML</li> 
+                        </ul>
+                        <div class="btn-container">
+                            <button class="see-project-btn" type="button">See project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="heading">
+              <h1 class="primary color1">About me</h1>  
```

Comment:
As mentioned above in the `My Recent Works` section, kindly I suggest you use the `h2` tag instead of the `h1` tag for the sub-heading for best practices. 😄